### PR TITLE
Update to dropwizard 1.0.0

### PR DESCRIPTION
**Pull request by craigching**
Note that consul-client shades it's dependencies except guava, so I removed some excludes that aren't relevant anymore.

### CODE REVIEW

File: example-app/dependency-reduced-pom.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <parent>
     <artifactId>dw-consul</artifactId>
     <groupId>com.boundary.dropwizard.consul</groupId>
-    <version>0.16-SNAPSHOT</version>
+    <version>0.18-SNAPSHOT</version>
```
Comment (gjesse) [inline example-app/dependency-reduced-pom.xml]: actually this file got into source by accident - it shouldn't be included at all and `dependency-reduced-pom.xml` should be in .gitignore. Can you go ahead and delete it?

Comment (craigching) [inline example-app/dependency-reduced-pom.xml]: yep!  I wondered about that ;)

File: pom.xml [hunk 2]
```diff
@@ -34,9 +34,9 @@
 
     <properties>
         <guava.version>18.0</guava.version>
-        <dropwizard.version>0.9.2</dropwizard.version>
-        <orbitz.consul.version>0.11.3</orbitz.consul.version>
-        <jackson.version>2.6.3</jackson.version>
+        <dropwizard.version>1.0.0</dropwizard.version>
```
Comment (gjesse) [inline pom.xml:36]: guava.version looks unused - it should pick up from the dw bom

File: example-app/dependency-reduced-pom.xml (hunk 1 above)
Comment (gjesse) [inline example-app/dependency-reduced-pom.xml]: yeah i threw this together quickly on the weekend :/

File: pom.xml (hunk 2 above)
Comment (craigching) [inline pom.xml:36]: I thought you didn't want the dw bom for this one?

Comment (gjesse) [inline pom.xml:36]: oh yeah, you're right. thanks.

**Pull request merged**